sdks/python/apache_beam/testing/load_tests/pardo_test.py

```diff
@@ -132,22 +137,38 @@
 @unittest.skipIf(not load_test_enabled, 'Enabled only for phrase triggering.')
 class ParDoTest(LoadTest):
   def setUp(self):
-    self.output = self.pipeline.get_option('output')
-    self.iterations = self.pipeline.get_option('number_of_counter_operations')
+    super(ParDoTest, self).setUp()
+    self._apply_filter()
+    self.iterations = self._get_option_or_default('iterations')
+    self.nb_of_counter = self._get_option_or_default('number_of_counters')
+    self.nb_of_operations = self._get_option_or_default(
+        'number_of_counter_operations')
+
+  def _apply_filter(self):
+    """Prevents metrics from namespace other than specified in pipeline
+    options from being published."""
+    if self.metrics_monitor is not None:
+      self.metrics_monitor.filters = MetricsFilter().with_namespace(
+          self.metrics_namespace)
+
+  def _get_option_or_default(self, opt_name, default=1):
```

Comment:
Should the `1` actually a default value? Maybe it would be better to have 0 and then omit loops as default?

Reply:
All options will be overridden in a Jenkins job anyway. But I think it makes sense to skip loops as default.